test/bats/plugin-test.bats

```diff
@@ -295,6 +295,34 @@ SLEEP_TIME=1
     assert_success
 }
 
+@test "verifier crd status check" {
+    teardown() {
+        echo "cleaning up"
+        wait_for_process ${WAIT_TIME} ${SLEEP_TIME} 'kubectl delete verifiers.config.ratify.deislabs.io/verifier-license-checker'
+    }
+
+    # apply a valid verifier, validate status property shows success
+    run kubectl apply -f ./config/samples/clustered/verifier/config_v1beta1_verifier_complete_licensechecker.yaml
+    assert_success
+    run bash -c "kubectl describe verifiers.config.ratify.deislabs.io/verifier-license-checker -n ${RATIFY_NAMESPACE} | grep 'Issuccess:  true'"
```

Comment:
Looks like the test is failing. It seems it's because the `$RATIFY_NAMESPACE` does not exist. I think you should solve this by adding this variable at the top: `RATIFY_NAMESPACE=gatekeeper-system` like base-test.bats has

Reply:
thx fixed now